### ISSUE TITLE
Add New Wallet - problem with connected. 

### ISSUE THREAD

**Issue by wronka1007** [emeth-defi-automation/portfolio-management-platform#131]
<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:x="urn:schemas-microsoft-com:office:excel"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Excel.Sheet>
<meta name=Generator content="Microsoft Excel 15">
<link id=Main-File rel=Main-File
href="file:///C:/Users/aleks/AppData/Local/Temp/msohtmlclip1/01/clip.htm">
<link rel=File-List
href="file:///C:/Users/aleks/AppData/Local/Temp/msohtmlclip1/01/clip_filelist.xml">
<style>

</style>
</head>

<body link="#0563C1" vlink="#954F72">

Description | Steps | Expected Results | Current results | Comments
-- | -- | -- | -- | --
Add   New Wallet | 1) Open dashboard     2) Go to the wallets tab     3) Click "Add New Wallet"     4) Choose Type Executable     5) Click "Connect Wallet"     6) Click next to the selection form without selecting any wallet | No connected wallets available | Wallet merges even though no   wallet is selected from the list. Network not selectable, entering wallet   name has no effect on wallet connection | The situation does not happen   every time, it usually occurs after the reproduction steps have been   passed

</body>

</html>

**Comment by Uacias** [comment]
Good point, this is known issue due to 'weird' behavior of configs in the app. 
We are currently looking for solution to fix this, possibly gonna need to rebuild config structure in our app.

Let me know if I'm on point with that guys
@neotheprogramist  @pgajek
